src/Event/ModelEventsHandler.php

```diff
@@ -89,9 +92,13 @@ private function ignoreModelEvents(Table $table): void
     {
         foreach (self::$ormEvents as $ormEvent) {
             foreach ($table->getEventManager()->listeners($ormEvent) as $listeners) {
-                if (array_key_exists('callable', $listeners) && is_array($listeners['callable'])) {
-                    foreach ($listeners['callable'] as $listener) {
-                        $this->processListener($table, $listener, $ormEvent);
+                if (array_key_exists('callable', $listeners)) {
+                    try {
+                        $reflection = new ReflectionFunction($listeners['callable']);
+                        $obj = $reflection->getClosureThis();
+                        $this->processListener($table, $obj, $ormEvent);
```

Comment:
Reflection was the only way one can get an object from a Closure (as far as I know at least 😅 )

Reply:
Moved this to the processListener method since in there the distinction between table, behaviors and other listeneres is made